internal/host/hostcommands/install_cmd.go

```diff
@@ -288,10 +288,16 @@ func appendDHCPArgs(cluster *common.Cluster, installerArgs []string, log logrus.
 		return append(installerArgs, "--append-karg", "ip=dhcp"), nil
 	}
 
-	if swag.StringValue(cluster.Kind) == models.ClusterKindAddHostsCluster {
-		return installerArgs, nil
+	if swag.StringValue(cluster.Kind) != models.ClusterKindAddHostsCluster {
+		return installerArgs, errors.Errorf("cannot determine machine network address family")
+	}
+
+	if v4, v6, err := hostutil.GetAddressFamilies(host); err != nil {
+		return installerArgs, err
+	} else if v4 && v6 {
+		log.Warnf("Cannot set DHCP kernel argument for host %s of day-2 cluster %s with dual IP stack. Not doing so may result in failing to download ignition or ISO", host.ID, *cluster.ID)
```

Comment:
i think that in case v4 and v6 we should print warning and return, but otherwise we can determine which argument to add to the kernel args

Reply:
good point, will fix and add tests